docs/concepts/queries.md

```diff
@@ -0,0 +1,92 @@
+There are three kinds of queries in ksqlDB: persistent, push, and pull.
```

Comment:
the build is flagging two broken links: 

As a dual to the [push query](push.md) example, a pull query for a geographic...

For asynchronous application flows, see
[Push Query](push.md).